specs.wdio/browser_sdk/quota.spec.ts

```
@@ -17,7 +18,7 @@ describe('quota', () => {
   afterEach(Session.afterEach);
   after(Session.deleteBrowsers);
 
-  it('is possible to retrieve the quota information of a user', async () => {
+  it.only('is possible to retrieve the quota information of a user', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)